src/v2/view-builder/views/phone/EnrollAuthenticatorPhoneView.js

```diff
@@ -1,173 +1,43 @@
-import { _, loc, Model } from 'okta';
```

Comment:
Moved this to `EnrollAuthenticatorDataPhoneView` and using this for the `Enter Code` purpose now.